### DISCUSSION TITLE
ShellBar Search Field Placement

### DISCUSSION THREAD

**Discussion by kagos** [UI5/webcomponents-react#4237] [Q&A]
Per the Fiori Horizon specifications outlined in the Visual Core wiki, the search field is centered for XL screens. Does the UI5 WC ShellBar component support the search position change?

https://wiki.one.int.sap/wiki/pages/viewpage.action?pageId=2697898394

**Comment by MarcusNotheis** [comment]
Hi @kagos 
I've moved this question to https://github.com/SAP/ui5-webcomponents/discussions/6593 🙂

**Comment by Lukas742** [comment]
Closing this discussion as it's a duplicate of https://github.com/SAP/ui5-webcomponents/discussions/6593
